eng/pipelines/coreclr/ilasm.yml

```diff
@@ -1,5 +1,14 @@
 trigger: none
 
+pr:
```

Comment:
I presume this shouldn't affect anyone's ability to manually trigger the pipeline in any branch for any PR.